// ng2-bs3-modal/components/modal-body.ngfactory.js
/**
 * This file is generated by the Angular 2 template compiler.
 * Do not edit.
 */
/* tslint:disable */
var __extends = (this && this.__extends) || function (d, b) {
    for (var p in b) if (b.hasOwnProperty(p)) d[p] = b[p];
    function __() { this.constructor = d; }
    d.prototype = b === null ? Object.create(b) : (__.prototype = b.prototype, new __());
};
import * as import1 from '@angular/core/src/linker/view';
import * as import2 from '@angular/core/src/linker/element';
import * as import3 from 'ng2-bs3-modal/components/modal-body';
import * as import4 from '@angular/core/src/linker/view_utils';
import * as import6 from '@angular/core/src/linker/view_type';
import * as import7 from '@angular/core/src/change_detection/change_detection';
import * as import8 from '@angular/core/src/metadata/view';
import * as import9 from '@angular/core/src/linker/component_factory';
var renderType_ModalBodyComponent_Host = null;
var _View_ModalBodyComponent_Host0 = (function (_super) {
    __extends(_View_ModalBodyComponent_Host0, _super);
    function _View_ModalBodyComponent_Host0(viewUtils, parentInjector, declarationEl) {
        _super.call(this, _View_ModalBodyComponent_Host0, renderType_ModalBodyComponent_Host, import6.ViewType.HOST, viewUtils, parentInjector, declarationEl, import7.ChangeDetectorStatus.CheckAlways);
    }
    _View_ModalBodyComponent_Host0.prototype.createInternal = function (rootSelector) {
        this._el_0 = this.selectOrCreateHostElement('modal-body', rootSelector, null);
        this._appEl_0 = new import2.AppElement(0, null, this, this._el_0);
        var compView_0 = viewFactory_ModalBodyComponent0(this.viewUtils, this.injector(0), this._appEl_0);
        this._ModalBodyComponent_0_4 = new import3.ModalBodyComponent();
        this._appEl_0.initComponent(this._ModalBodyComponent_0_4, [], compView_0);
        compView_0.create(this._ModalBodyComponent_0_4, this.projectableNodes, null);
        this.init([].concat([this._el_0]), [this._el_0], [], []);
        return this._appEl_0;
    };
    _View_ModalBodyComponent_Host0.prototype.injectorGetInternal = function (token, requestNodeIndex, notFoundResult) {
        if (((token === import3.ModalBodyComponent) && (0 === requestNodeIndex))) {
            return this._ModalBodyComponent_0_4;
        }
        return notFoundResult;
    };
    return _View_ModalBodyComponent_Host0;
}(import1.AppView));
function viewFactory_ModalBodyComponent_Host0(viewUtils, parentInjector, declarationEl) {
    if ((renderType_ModalBodyComponent_Host === null)) {
        (renderType_ModalBodyComponent_Host = viewUtils.createRenderComponentType('', 0, import8.ViewEncapsulation.None, [], {}));
    }
    return new _View_ModalBodyComponent_Host0(viewUtils, parentInjector, declarationEl);
}
export var ModalBodyComponentNgFactory = new import9.ComponentFactory('modal-body', viewFactory_ModalBodyComponent_Host0, import3.ModalBodyComponent);
var styles_ModalBodyComponent = [];
var renderType_ModalBodyComponent = null;
var _View_ModalBodyComponent0 = (function (_super) {
    __extends(_View_ModalBodyComponent0, _super);
    function _View_ModalBodyComponent0(viewUtils, parentInjector, declarationEl) {
        _super.call(this, _View_ModalBodyComponent0, renderType_ModalBodyComponent, import6.ViewType.COMPONENT, viewUtils, parentInjector, declarationEl, import7.ChangeDetectorStatus.CheckAlways);
    }
    _View_ModalBodyComponent0.prototype.createInternal = function (rootSelector) {
        var parentRenderNode = this.renderer.createViewRoot(this.declarationAppElement.nativeElement);
        this._text_0 = this.renderer.createText(parentRenderNode, '\n        ', null);
        this._el_1 = this.renderer.createElement(parentRenderNode, 'div', null);
        this.renderer.setElementAttribute(this._el_1, 'class', 'modal-body');
        this._text_2 = this.renderer.createText(this._el_1, '\n            ', null);
        this.renderer.projectNodes(this._el_1, import4.flattenNestedViewRenderNodes(this.projectableNodes[0]));
        this._text_3 = this.renderer.createText(this._el_1, '\n        ', null);
        this._text_4 = this.renderer.createText(parentRenderNode, '\n    ', null);
        this.init([], [
            this._text_0,
            this._el_1,
            this._text_2,
            this._text_3,
            this._text_4
        ], [], []);
        return null;
    };
    return _View_ModalBodyComponent0;
}(import1.AppView));
export function viewFactory_ModalBodyComponent0(viewUtils, parentInjector, declarationEl) {
    if ((renderType_ModalBodyComponent === null)) {
        (renderType_ModalBodyComponent = viewUtils.createRenderComponentType('C:/Users/alex/WebstormProjects/footbalQuiz/node_modules/ng2-bs3-modal/components/modal-body.d.ts class ModalBodyComponent - inline template', 1, import8.ViewEncapsulation.None, styles_ModalBodyComponent, {}));
    }
    return new _View_ModalBodyComponent0(viewUtils, parentInjector, declarationEl);
}
